### PR TITLE
Intimidation threshold, learn from mistakes, fixes #40

### LINKED ISSUE #40: Clarify the intimidation threshold
On discussing with @cjacoby, PI's top takeaways could be further clarified.

Overcome the intimidation threshold by:
1. Have a structure to make it easy to share work
2. Role-model sharing work
3. Make it non-optional to share, insist the structure is used.

### PR DESCRIPTION
**Pull request by ewallace**

### CODE REVIEW

File: open-science-projects.md [hunk 1]
```diff
@@ -224,7 +224,9 @@ Students also learn about the process of writing a collaborative paper by attend
 ## Mistakes and benefits
 
 We have made plenty of mistakes.
-Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented) data structures.
+Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented and barely usable) data structures.
+Offering related projects over subsequent years helped us to learn from our previous mistakes, and to make new ones instead.
```
Comment (FlicAnderson) [inline open-science-projects.md:228]: Maybe would reword this to "helped us to learn from our previous mistakes, and PROGRESS TO new ones instead." - it gives a better sense of forward progress than an immovable brick wall of mistakes preventing progression?

File: open-science-projects.md [hunk 2]
```diff
@@ -224,7 +224,9 @@ Students also learn about the process of writing a collaborative paper by attend
 ## Mistakes and benefits
 
 We have made plenty of mistakes.
-Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented) data structures.
+Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented and barely usable) data structures.
```
Comment (FlicAnderson) [inline open-science-projects.md:227]: Perhaps "undocumented and difficult to use"?

**Review by FlicAnderson — COMMENTED** [review]
Nice fix to make PI takeaways more grammatically coherent & active! 

I'd recommend light wording tweaks to avoid giving an overly negative impression from the mistakes section, as tone is difficult to convey with something like this?

File: open-science-projects.md (hunk 1 above)
Comment (ewallace) [reply open-science-projects.md:228]: Yes please!

File: open-science-projects.md (hunk 2 above)
Comment (ewallace) [reply open-science-projects.md:227]: Yes, or we could go back to just “undocumented”

File: open-science-projects.md (hunk 1 above)
Comment (ewallace) [reply open-science-projects.md]: edited.

**Comment by ewallace** [comment]
Thanks Flic, that helps. Can we merge with my new update?

**Review by FlicAnderson — APPROVED** [review]
